### PR TITLE
Add meta package, and add authors to each contained packages

### PR DESCRIPTION
**Pull request by 130s**
Discussed in #31.
@sachinchitta @rethink-imcmahon

### CODE REVIEW

File: baxter/baxter_moveit_config/package.xml [hunk 1]
```diff
@@ -5,8 +5,9 @@
   <description>
      An automatically generated package with all the configuration and launch files for using the baxter with the MoveIt Motion Planning Framework
   </description>
-  <author email="assistant@moveit.ros.org">MoveIt Setup Assistant</author>
+  <author>Rethink Robotics Inc.</author>
   <maintainer email="assistant@moveit.ros.org">MoveIt Setup Assistant</maintainer>
+  <maintainer email="rsdk.support@rethinkrobotics.com">Rethink Robotics Inc.</maintainer>
```
Comment (rethink-imcmahon) [inline baxter/baxter_moveit_config/package.xml]: Just a bit more metadata:
```
+  <url type="website">http://sdk.rethinkrobotics.com/wiki/MoveIt_Tutorial</url>
+  <url type="bugtracker">https://github.com/ros-planning/moveit_robots/issues</url>
+  <url type="repository">https://github.com/ros-planning/moveit_robots</url>
```

Comment (rethink-imcmahon) [inline baxter/baxter_moveit_config/package.xml]: Though this is really a note for me to update the `package.xml`. I can add a commit separately if you prefer.

**Comment by rethink-imcmahon** [comment]
+1 great work on this @130s. Looks good to me!

File: baxter/baxter_moveit_config/package.xml (hunk 1 above)
Comment (130s) [inline baxter/baxter_moveit_config/package.xml]: Updated URLs.

**Pull request merged**